test/labeling/test_convergence.py

```diff
@@ -77,15 +78,27 @@ def test_labeling_convergence(self) -> None:
             + [get_positive_labeling_function(divisor) for divisor in range(2, 9)]
             + [get_negative_labeling_function(divisor) for divisor in range(2, 9)]
         )
+        logging.info("CREATED LABELING FUNCTIONS")
```

Comment:
What report at both info and debug level?

Reply:
Removed